core/channel/async_migrate_channel_unittest.cpp

```diff
@@ -104,11 +96,11 @@ TEST_F(TestAsyncMigrateChannel, MigrateOtherIncProgress) {
     ObjList<Obj> obj_list;
 
     auto& scr_int = obj_list.create_attrlist<int>("int");
-    auto& src_attr = obj_list.create_attrlist<Attr>("attr");
+    auto& src_attr = obj_list.create_attrlist<std::string>("attr");
```

Comment:
It looks like the root cause of the failure in my [PR](https://travis-ci.org/husky-team/husky/builds/171012980)?